### ISSUE TITLE
Add the list to the list of awesome lists :)

### ISSUE THREAD

**Issue by samdark** [samdark/awesome-yii#16]
https://github.com/sindresorhus/awesome/

Contribution guidelines are quite strict so some adjustments are required.

**Comment by cebe** [comment]
https://github.com/sindresorhus/awesome/blob/master/contributing.md

**Comment by iJackUA** [comment]
@samdark @cebe Guys, I will be happy to grant you rights to manage this repo. 
Currently, PHP is not the top prio of my interests and  I do not pay a lot of attention to this repo.
Do you know is there any ability to transfer repo to another organization/user?

**Comment by samdark** [comment]
Yes. I saw it's done but I'm not sure how...

**Comment by iJackUA** [comment]
Yes, I have found how to do this. But it says "samdark/awesome-yii already exists" :)
So delete your fork and I will transfer it to you.

**Comment by samdark** [comment]
Done.

**Comment by iJackUA** [comment]
You should get the confirmation request

**Comment by samdark** [comment]
Seems it worked. Thanks!

**Comment by fancyecommerce** [comment]
e-commerce shop : Fecshop
github: https://github.com/fecshop/yii2_fecshop
